tests/tests-core/src/main/java/com/gentics/mesh/core/field/s3binary/S3BinaryFieldEndpointTest.java

```diff
@@ -54,13 +63,26 @@ public class S3BinaryFieldEndpointTest extends AbstractFieldEndpointTest {
      */
     @Before
     public void updateSchema() throws IOException {
-        try (Tx tx = tx()) {
-            S3BinaryFieldSchema s3BinaryFieldSchema = new S3BinaryFieldSchemaImpl();
-            s3BinaryFieldSchema.setName(FIELD_NAME);
-            s3BinaryFieldSchema.setLabel("Some label");
-            prepareTypedSchema(schemaContainer("content"), List.of(s3BinaryFieldSchema), Optional.empty());
-            tx.success();
-        }
+    	HibSchema schemaContainer = schemaContainer("content");
```

Comment:
Here we use REST API to update the schema, because it triggers the indexing engine to get updated with the new field data.